simple_history/manager.py

```diff
@@ -91,6 +91,18 @@ def latest_of_each(self):
             )
         return latest_historics
 
+    def select_related_history_tracked_objs(self):
```

Comment:
> I'm not sure this is a great idea. I do see the utility in it, but it makes assumptions on the user's application. It's also something that the user should be able to do on their own. Once this gets merged, it becomes very difficult to remove or change to maintain backwards compatibility. It's easier to keep the library slim (if it can event be called that 😁) and provide hooks for the user to do what they need.

Good point, I agree 👍

> If this is necessary for the diffing logic, then I'd prefer if it were an undocumented method so we can change it as needed moving forward.

It's necessary to reduce the number of DB queries, but I see no problem in simply moving the code to [the only place it's currently used](https://github.com/raunaq-sailo/django-simple-history/blob/d054c33e75e32851cefc59c19ff2225e26d2808d/simple_history/admin.py#L132). Not sure whether I prefer removing the documentation or inlining the code in the previously linked method 🤔 Any opinions on those two alternatives?

Reply:
I like the separate function. It gets us closer to something similar to https://docs.djangoproject.com/en/5.0/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_select_related, but I don't feel strongly about it.

Reply:
True 👍
I'll prefix the method with `_` while I'm at it.